app/ide-desktop/lib/client/src/config/parser.ts

```diff
@@ -78,48 +92,48 @@ function printHelp(cfg: { args: config.Args; groupsOrdering: string[]; helpExten
         if (option.primary || cfg.helpExtended) {
             const cmdOption = naming.camelToKebabCase(optionName)
             maxOptionLength = Math.max(maxOptionLength, stringLength(cmdOption))
-            const entry = [cmdOption, option]
             const section = sections[option.name]
-            if (section != null) {
-                section.entries.unshift(entry)
+            if (section !== undefined) {
+                section.entries.unshift([cmdOption, option])
             } else {
-                topLevelSection.entries.push(entry)
+                topLevelSection.entries.push([cmdOption, option])
```

Comment:
Why this change? Refactoring the literal to entry made the code shorter and less repetitive.

Reply:
i removed `entry` since it requires either:
- a type annotation (longer, adds a tiny bit of maintenance cost), or
- `as const` - this is what it was previously, i just figured it'd be good to minimize usage of `as` (even `as const`) because `as T` (not `as const`) isn't quite type-safe.

thoughts?

Reply:
Makes sense. Ok, its up to you, whatever you like better :)